docs/manuals/core/config/config-checker.md

```diff
@@ -13,8 +13,8 @@ exception if at least one issue collected has the `ERROR` level.
 
 Here is the list of the possible issues that could be returned by the checker:
 
-- An `ERROR` issue is created if the `clean_entities_enabled` property is populated in the `GlobalAppConfig^` with a
-  non-Boolean value.
+- An `WARNING` issue is created if the `repository_type` property is populated in the `CoreSection^` with an
+  unsupported repository value.
 - An `ERROR` issue is created if the `storage_type` and the `scope` properties of any `DataNodeConfig^` have not
```

Comment:
An `ERROR` issue is created if the `storage_type` and the `scope` properties of any `DataNodeConfig^` are not